packages/snap/src/cloud/build/builders/node/index.ts

```diff
@@ -49,12 +49,18 @@ export class NodeBuilder implements StepBuilder {
         '// {{router paths}}',
         steps
           .map(
-            (step, index) =>
-              `'${step.config.method} ${step.config.path}': { stepName: '${step.config.name}', handler: route${index}.handler, config: route${index}.config }`,
+            (step, index) => {
+              this.listener.onBuildStart(step)
```

Comment:
we probably don't need this

Reply:
What do you recommend then, removing all individual api steps logs from the build logs?